### DISCUSSION TITLE
Running packer build on windows-2022.pkr.hcl

### DISCUSSION THREAD

**Discussion by michaelmowry** [actions/runner-images#9030] [General questions] [answered]
Hey all,

We would like to use packer to build the windows-2022 image from windows-2022.pkr.hcl. We made some minor modifications to the file to publish the image to a shared image gallery in Azure and the build appears to run fine until we hit the Miniconda issue below. It is unable to find the checksum and fails the build. Any suggestions on how we might fix this?

> ...
> ==> azure-arm.image: Provisioning with powershell script: ./../scripts/build/Install-Miniconda.ps1
> ==> azure-arm.image: Unable to find checksum for Miniconda3-latest-Windows-x86_64.exe in https://repo.anaconda.com/miniconda/
> ==> azure-arm.image: At C:\Windows\Temp\script-657a12b8-dc10-761e-9d14-2a04bb92e5fc.ps1:21 char:5
> ==> azure-arm.image: +     throw "Unable to find checksum for $installerName in https://repo/ ...
> ==> azure-arm.image: +     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> ==> azure-arm.image:     + CategoryInfo          : OperationStopped: (Unable to find ....com/miniconda/:String) [], RuntimeException
> ==> azure-arm.image:     + FullyQualifiedErrorId : Unable to find checksum for Miniconda3-latest-Windows-x86_64.exe in https://repo.anacond/
> ==> azure-arm.image:    a.com/miniconda/
> ==> azure-arm.image:
> ==> azure-arm.image: Provisioning step had errors: Running the cleanup provisioner, if present...
> ...

**Comment by erik-bershel** [comment, marked as answer]
Hey @michaelmowry!
I assume that you are using a release branch or an outdated version of the main one to build. Using the latest version of the main branch of the repository to build will solve the problem.

**Comment by mayanknishad12** [comment]
Uploading 20260806_172233.mp4…

**Comment by harperedit** [comment]
Thanks for sharing this info! For anyone interested in casual card and mini-games on smartphone, getting the direct setup file is easy. You can complete the [Download](https://pk67gameapp.com/download/) process in a few clicks and follow the guide [For iOs](https://pk67gameapp.com/pk67-for-ios/) if you are using an iPhone.

**Comment by harperedit** [comment]
Great points made in this thread! WinPKR is one of the smooth gaming apps for mobile devices. You can visit the official [Home](https://winpkrplay.com.pk/) portal to learn more and perform the [Winpkr Download](https://winpkrplay.com.pk/download/) to start playing.

**Comment by harperedit** [comment]
Very informative discussion! For anyone looking to install the application, click on [Winpkr Download](https://winpkrplay.com.pk/download/) to get the official APK file, and visit the main [Home](https://winpkrplay.com.pk/) screen for updates.

**Comment by harperedit** [comment]
Thanks for sharing this info! For anyone interested in casual card and mini-games on smartphone, getting the direct setup file is easy. You can complete the [Download](https://pk67gameapp.com/download/) process in a few clicks and follow the guide [For iOs](https://pk67gameapp.com/pk67-for-ios/) if you are using an iPhone.
